Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -277,6 +277,16 @@
   - type: Insulated
   - type: Fiber
     fiberMaterial: fibers-insulative
+  - type: MeleeWeapon #Sunrise-start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)